Toggle between standard and satellite views in OBASearchResultsMapVC

ualch9 opened the pull request:
Fixes #65 - more map options (satellite, 3d, etc.)

aaronbrethorst commented on OBAKit/OBAApplication.m:
```diff
@@ -93,6 +93,7 @@ - (void)registerAppDefaults {
     defaults[kUngroupedBookmarksOpenKey] = @(YES);
     defaults[OBAOptInToTrackingDefaultsKey] = @(YES);
     defaults[OBAAllowReviewPromptsDefaultsKey] = @(YES);
+    defaults[OBAMapSelectedTypeDefaultsKey] = MKMapTypeStandard;
```
I'm guessing that this works only because `MKMapTypeStandard` is equal to `0`, which the compiler is interpreting as nil. Please box this. i.e. `@(MKMapTypeStandard)`

ualch9 commented:
For `credits.html`, should I list every single icon used `Direction - created by Ricardo Cardoso` or can I just use `This project also includes Material Icons by Google Inc. licensed under the terms of Apache 2.0`

barbeau commented:
On Android we include attributions for all icon sources.  Where did the
Ricardo icon come from? Sources outside of official Google material design
icons may have specific attribution requirements.

On Jan 8, 2017 3:50 PM, "Alan Chu" <notifications@github.com> wrote:

For credits.html, should I list every single icon used Direction - created
by Ricardo Cardoso or can I just use This project also includes Material
Icons by Google Inc. licensed under the terms of Apache 2.0

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/OneBusAway/onebusaway-iphone/pull/931#issuecomment-271178430>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AA4pLexUta_KYtmjt_lQ0z2UKzdPcbCnks5rQUwdgaJpZM4LdpQy>
.

ualch9 commented:
OK, just following Google's attribution requirements should be fine then

barbeau commented:
Just be careful about the source - for example,
https://materialdesignicons.com is a mix of Google icons and those created
by individuals.  If the icon came from here you will need to figure out if
Google or someone else was the author.  If a name was attached to the logo
then most likely that was the author.

The site https://material.io/icons/ is the canonical Google material design
icon source. If all icons came from here then just the Google attribution
is fine.

On Jan 8, 2017 7:02 PM, "Alan Chu" <notifications@github.com> wrote:

> OK, just following Google's attribution requirements should be fine then
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OneBusAway/onebusaway-iphone/pull/931#issuecomment-271190501>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA4pLZlafCVGrsdxngXgTGwpf97QHwYcks5rQXj8gaJpZM4LdpQy>
> .
>

ualch9 commented:
Fixed per your feedback `9dbf6fa`
<img src="https://cloud.githubusercontent.com/assets/22162410/21756919/2bc0af4c-d5dc-11e6-9637-a73f493a4c8e.png" height=480>

barbeau commented:
Also, I believe material design icons from Google are under CC BY 4.0:
https://creativecommons.org/licenses/by/4.0/

aaronbrethorst commented on OneBusAway/credits.html:
```diff
@@ -366,6 +366,7 @@ <h2 id='ptezdashboard'>Lumberjack Console</h2>
       <li><a href='https://thenounproject.com/term/update/698616'>Update</a> - created by Ricardo Cardoso</li>
       <li><a href='https://thenounproject.com/term/warning/33183/'>Warning</a> - created by Milky - Digital innovation, FR</li>
     </ul>
+    <p>This project also includes Material Icons by Google Inc. licensed under the terms of Apache 2.0</p>
```
Per @barbeau, this needs to be updated.

ualch9 replied:
https://github.com/google/material-design-icons/blob/master/LICENSE

aaronbrethorst replied:
Ah, fair enough.

ualch9 replied:
It looks like they [made the change](https://github.com/google/material-design-icons/commit/68e015dbbb6b730b5fe4934e8507cd5a465c8a3d) fairly recently

aaronbrethorst commented on OneBusAway/controls/controls/OBAFloatingButton.swift:
```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+@IBDesignable
+open class OBAFloatingButton: UIButton {
```
I like the idea of having a FAB, but shouldn't it have a drop shadow?

ualch9 commented:
Updated `e4ee8b4`

ualch9 commented:
It looks like the build failed, fixing...

ualch9 commented:
Fixed and updated `5d7a1d5`
<img src="https://cloud.githubusercontent.com/assets/22162410/21794340/5b3241c8-d6ae-11e6-8118-b951abb53fa9.png" height=480>

aaronbrethorst commented:
Awesome, thank you!
